uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/prebookconfirmation/PrebookConfirmationView.kt

```diff
@@ -41,7 +41,7 @@ class PrebookConfirmationView @JvmOverloads constructor(
         defStyleAttr: Int = 0)
     : FrameLayout(context, attrs, defStyleAttr), ScheduledDateView {
     private val scheduledDateViewBinder = ScheduledDateViewBinder()
-    var actions: BookingRequestMVP.Actions? = null
+    var actions: BookingRequestContract.Actions? = null
```

Comment:
could this be private?

Reply:
It is used in the BookingActivity, cannot without some extra changes

Reply:
We should create a contract for the prebookConfirmationView and have a binding method for the actions that's called from the BookingActivity